tests/web3js/debug_traces_test.js

```diff
@@ -585,15 +585,15 @@ it('should retrieve call traces', async () => {
         {
             from: conf.eoa.address.toLowerCase(),
             gas: '0xcdd4',
-            gasUsed: '0xb38f',
+            gasUsed: '0xb3ed',
             to: contractAddress.toLowerCase(),
             input: '0xc550f90f',
             output: '0x0000000000000000000000000000000000000000000000000000000000000007',
             calls: [
                 {
                     from: contractAddress.toLowerCase(),
                     gas: '0x6d44',
-                    gasUsed: '0x524a',
+                    gasUsed: '0x52a8',
```

Comment:
Change comes from: `EIP-7623: Increase calldata cost`.
See: https://mixbytes.io/blog/the-prague-electra-pectra-hardfork-explained#rec860520137 .